pulse2percept/__init__.py

```diff
@@ -1,29 +1,32 @@
 from __future__ import absolute_import, division, print_function
 from .version import __version__  # noqa
-from .api import *
-from . import retina
-from . import implants
-from . import stimuli
-from . import files
-from . import utils
-
 import logging
 
-
 # Disable Jupyter Notebook handlers
 # https://github.com/ipython/ipython/issues/8282
 logging.getLogger().handlers = []
 
-# Set up root logger
+# Set up root logger for debug file
 formatstr = '%(asctime)s [%(name)s] [%(levelname)s] %(message)s'
 logging.basicConfig(level=logging.DEBUG,
                     format=formatstr,
                     filename='debug.log',
                     filemode='w')
-logging.getLogger(__name__).info('Welcome to pulse2percept')
 
-# Add streaming to console
+# Add streaming to console: Temporarily set level to ERROR so the imports
+# don't trigger 'deprecated' warnings
 console = logging.StreamHandler()
-console.setLevel(logging.INFO)
+console.setLevel(logging.ERROR)

```

Comment:
Will do the re-render.

The `deprecated` decorator is automatically executed upon module import. This is just [how decorators work](http://stackoverflow.com/questions/18062443/python-decorator-function-called-at-compile-time). An unfortunate side effect is that all these warnings are generated...
An easy work-around is to temporarily set the logging level higher, so the warnings will not print to console. There are [more sophisticated solutions](http://stackoverflow.com/questions/19601468/how-to-make-python-decorator-not-run-when-imported), but I feel like I've wasted enough time already trying to wrap my head around nested decorators.